test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -940,7 +940,7 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 
 						var expectedReadyNodes int
 						if !eng.ExpandedDefinition.Properties.HasLowPriorityScaleset() {
-							expectedReadyNodes = eng.NodeCount()
```

Comment:
By setting this expected count against the value in the api model, we don't take into account that the list we enumerate through in line 918 above may not have the exact same numbers due to <some> environmental behaviors. Because we are specifically testing docker crash simulation in this test, we want to make sure the node count before and after operate against a common number of nodes.